index.html

```diff
@@ -34,6 +34,35 @@
         <a href="#about"><p>About</p></a>
         <a href="#contact"><p>Contact</p></a>
       </nav>
+    </div>
+    <div id="overlay">s
+      <div id="popup">
+        <div class="popup-header">
+          <h4 id="popup-project-heading" class="popup-heading">This Heading</h4>
+          <i id="close-btn" href="#" onclick="PopupWindowOff()" class="fa-solid fa-close fa-2x close-popup"></i>
+        </div>
+        <div class="popup-body">
+          <div class="popup-image-area">
+            <img id="popup-project-image" alt="" />
+          </div>
+          <div class="popup-detail-area">
+            <p id="popup-project-desc"></p>
+          </div>
+         <div class="popup-tech-area">
+          <ul id="popup-technologies" class="lists">
+            <li class="popup-list" id="popup-tech1"></li>
+            <li class="popup-list" id="popup-tech2"></li>
+            <li class="popup-list" id="popup-tech3"></li>
+            <li class="popup-list" id="popup-tech4"></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Please make sure that the list of technologies has the same design, paddings margins color and separators: 

Your design  | Figma design
------------- | -------------
<img width="325" alt="Screenshot 2022-09-09 at 15 53 39" src="https://user-images.githubusercontent.com/77038610/189354452-141893a2-df75-432c-b625-d2719ef48cea.png">  | <img width="377" alt="Screenshot 2022-09-09 at 15 53 20" src="https://user-images.githubusercontent.com/77038610/189354390-c698495f-4965-4c39-9347-08e65575dc6c.png">